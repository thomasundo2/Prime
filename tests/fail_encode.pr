int main()
{
    string x;
    x = "fail";
    /* encode(34l, x); UNDEFINED: set but can't reference */
    encode(34l);
}