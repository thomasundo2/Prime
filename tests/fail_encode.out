Fatal error: exception Failure("illegal argument lint expected string in 34")
